schematics/ng-add/index.ts

```diff
@@ -0,0 +1,53 @@
+import {Rule, SchematicContext, SchematicsException, Tree} from '@angular-devkit/schematics';
+import {
+  NodePackageInstallTask,
+  RunSchematicTask,
+} from '@angular-devkit/schematics/tasks';
+
+import {getWorkspace} from '@schematics/angular/utility/config';
+import {getProject} from '@schematics/angular/utility/project';
+
+import {Schema} from './schema';
+import * as messages from './messages';
+import {addPackageToPackageJson, getPackageVersionFromPackageJson} from '../utils/package-config';
+
+
+const NG_BOOTSTRAP_VERSION = '^6.0.0';
+const BOOTSTRAP_VERSION = '^4.4.0';
+
+/**
+ * This is executed when `ng add @ng-bootstrap/ng-bootstrap` is run.
+ * It installs all dependencies in the 'package.json' and runs 'ng-add-setup-project' schematic.
+ */
+export default function ngAdd(options: Schema): Rule {
+  return (tree: Tree, context: SchematicContext) => {
+
+    // Checking that project exists
+    const {project} = options;
+    if (project) {
+      const workspace = getWorkspace(tree);
+      const projectWorkspace = getProject(workspace, project);
+
+      if (!projectWorkspace) {
+        throw new SchematicsException(messages.noProject(project));
+      }
+    }
+
+    // Installing dependencies
+    const angularCoreVersion = getPackageVersionFromPackageJson(tree, '@angular/core') !;
+    const angularLocalizeVersion = getPackageVersionFromPackageJson(tree, '@angular/localize');
+
+    addPackageToPackageJson(tree, '@ng-bootstrap/ng-bootstrap', `${NG_BOOTSTRAP_VERSION}`);
+    addPackageToPackageJson(tree, 'bootstrap', `${BOOTSTRAP_VERSION}`);
```

Comment:
Why do we need `` `${ }` `` here ? It seems useless to me, both versions are already strings

Reply:
yep, it was `^${BOOTSTRAP_VERSION}` initially